misc/benchmark/nydusd_config.json

```diff
@@ -0,0 +1,30 @@
+{
+    "device": {
+        "backend": {
+            "type": "registry",
+            "config": {
+                "scheme": "http",
+                "host": "localhost:5000"
+            }
+        },
+        "cache": {
+            "type": "blobcache",
+            "config": {
+                "work_dir": "/var/lib/nydus/cache/"
+            }
+        }
+    },
+    "mode": "direct",
+    "digest_validate": false,
+    "iostats_files": true,
```

Comment:
`iostats_files`, `access_pattern`, `latest_read_files`

These would affect runtime performance, have we only enabled these in the access file list analysis and disable them in our benchmark tests?

Reply:
> `iostats_files`, `access_pattern`, `latest_read_files`
> 
> These would affect runtime performance, have we only enabled these in the access file list analysis and disable them in our benchmark tests?

Yes, we can move these configs to the runtime, it will work when we need.